clang/test/CodeGen/rounding-math.cpp

```diff
@@ -11,3 +11,55 @@ float V3 = func_01(1.0F, 2.0F);
 // CHECK: @V1 = {{.*}}global float 1.000000e+00, align 4
 // CHECK: @V2 = {{.*}}global float 1.000000e+00, align 4
 // CHECK: @V3 = {{.*}}global float 3.000000e+00, align 4
+
+void test_builtin_elementwise_fma_round_upward() {
```

Comment:
@arsenm Tests for different rounding modes. I chose this file rather than `constant-builtins-vector.cpp` because, it's already testing rounding modes and already has the appropriate `RUN`  command set up.